src/yarpmanager/src-manager/clusterWidget.cpp

```diff
@@ -477,14 +485,28 @@ bool ClusterWidget::checkNode(const string &name)
         portname = "/" + portname;
     }
 
+    if (!NetworkBase::checkNetwork(2.0))
+    {
+        yError()<<"ClusterWidget: yarpserver is not running";
+        return false;
+    }
+
+    yarp::profiler::NetworkProfiler::PortDetails dummy;
+    if (! yarp::profiler::NetworkProfiler::getPortDetails(portname, dummy))
+    {
+        yError()<<"ClusterWidget: port"<<portname<<"is not responding";
```

Comment:
`port"<<portname<<"is` --> `port "<<portname<<" is`?

Reply:
`yError`,`yInfo`, `yWarning` add automatically the spaces :wink:

Reply:
I had **no idea** of this.